app/uk/gov/hmrc/agentsubscriptionfrontend/controllers/package.scala

```diff
@@ -40,11 +42,13 @@ package object controllers {
       }
     }
 
-    private val nonEmptyTelephoneNumber: Constraint[String] = Constraint[String] { fieldValue: String =>
```

Comment:
Yes I have but it wasn't much of a change I just took out one of the case matches in the constraint. If I change it back to nonEmptyTelephoneNumber I will have to come up with different names for the other variables. I think the current name reflects what it does as well as keeping consistency.